src/js/components/CosmosErrorMessage.js

```diff
@@ -79,11 +73,14 @@ class CosmosErrorMessage extends React.Component {
   }
 
   render() {
+    let {className, onResized, wrapperClass} = this.props;
+
     return (
-      <div className={this.props.wrapperClass}>
+      <div className={wrapperClass}>
         {this.getHeader()}
         <CollapsibleErrorMessage
-          onToggle={this.props.onResized}
+          className={className}
```

Comment:
@wavesoft This broke some integration tests. Please run them on your PRs until we have them running again on CI. We have all been bad at this lately.